scripts/main.js

```diff
@@ -108,6 +108,7 @@ function renderAppDetail (id) {
 function renderAppList () {
   $body.attr('data-state', 'dashboard')
 
+  $appList.empty()
   applist.findAll()
     .then(function (apps) {
       apps.forEach(function (app) {
```

Comment:
Best is to comment it out, then create an app, go back, create another app, go back, without reloading the page, you’ll see :) The probem is that we use the `.append()` method, so we only add to what already exists. So if we don’t remove the current content, we will add the existing apps each time we navigate to the dashboard